eng/pipelines/templates/steps/archetype-sdk-docs.yml

```diff
@@ -0,0 +1,133 @@
+    workingDirectory: $(Build.BinariesDirectory)
+    displayName: Download and Extract DocFX
+  - pwsh: |
+      dotnet restore "$(Build.SourcesDirectory)/eng/docgeneration/assets/docgen.csproj" /p:BuildBinariesDirectory=$(Build.BinariesDirectory)
+    displayName: Restore Nuget Packages Needed
+  - task: DotNetCoreInstaller@2
+    displayName: "Use .NET Core sdk $(DotNetCoreSDKVersion)"
+    inputs:
+      version: $(DotNetCoreSDKVersion)
+  - ${{ if ne(variables['Build.DefinitionName'], 'net - client - ci')}}:
+    - ${{ each artifact in parameters.Artifacts }}:
+      - pwsh: |
+          echo "##vso[task.setvariable variable=packageLocation]${{parameters.ServiceDirectory}}/${{artifact.name}}"
+          echo "##vso[task.setvariable variable=artifactSafeName]${{artifact.safeName}}"
+          if ('${{parameters.ServiceDirectory}}' -eq '*') {
+            echo "##vso[task.setvariable variable=packageLocation]core/${{artifact.name}}"
+          }
+          if ('${{parameters.ServiceDirectory}}' -eq 'cognitiveservices') {
+            echo "##vso[task.setvariable variable=packageLocation]cognitiveservices/${{artifact.directoryName}}"
+            echo "##vso[task.setvariable variable=artifactSafeName]${{artifact.directoryName}}"
+          }
+          if ('${{parameters.LibType}}' -eq 'Management') {
+            echo "##vso[task.setvariable variable=packageLocation]${{artifact.serviceDir}}/${{artifact.name}}"
+            echo "##vso[task.setvariable variable=artifactSafeName]${{artifact.name}}"
+          }
+        displayName: Set Variables for current Loop
+      - pwsh: |
+          mkdir $(artifactSafeName)/dll-docs/my-api
+          mkdir $(artifactSafeName)/dll-docs/dependencies/my-api
+          mkdir $(artifactSafeName)/dll-xml-output
+          mkdir $(artifactSafeName)/dll-yaml-output
+          mkdir $(artifactSafeName)/docfx-output
+        displayName: Create Directories for Doc Generation
+        workingDirectory: $(Build.BinariesDirectory)
+      - pwsh: dotnet build eng/service.proj /p:ServiceDirectory=$(packageLocation) /p:IncludeTests=false /p:IncludeSamples=false /p:OutputPath=$(Build.BinariesDirectory)/$(artifactSafeName)/dll-docs/my-api /p:TargetFramework=netstandard2.0
+        displayName: Build Client Packages
+        condition: and(succeeded(), eq(False, '${{ parameters.LibType }}'))
+      - pwsh: dotnet build eng/service.proj /p:ServiceDirectory=$(packageLocation) /p:IncludeTests=false /p:IncludeSamples=false /p:OutputPath=$(Build.BinariesDirectory)/$(artifactSafeName)/dll-docs/dependencies/my-api /p:TargetFramework=netstandard2.0 /p:CopyLocalLockFileAssemblies=true
+        displayName: Build Client Packages, Include Dependencies
+        condition: and(succeeded(), eq(False, '${{ parameters.LibType }}'))
+      - script: dotnet msbuild eng/mgmt.proj /p:scope=$(packageLocation) /p:OutputPath=$(Build.BinariesDirectory)/$(artifactSafeName)/dll-docs/my-api -maxcpucount:1 -nodeReuse:false
```

Comment:
I believe you should be able to use the same build command for mgmt and data but you will have to pass it the scope and use the root build.proj.

Reply:
But am building only `netstandard2.0` for client libraries

Reply:
Filled issue to get this done.